src/internal/components/VisuallyHidden.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components'
```

Comment:
This is almost a duplicate of our existing internal [VisuallyHidden component](https://github.com/primer/react/blob/5a33122406f77a1eb186a198584fe5b8db77ccde/src/_VisuallyHidden.tsx). I like how this one's selector specifies not to apply this to focused/active elements.

Could we just update the existing component instead of introducing a new one?

Reply:
@mperrotti I can definitely take a look to see what the splash zone would be! `_VisuallyHidden` is reference in ~16 files from what I can tell and only a handful use the `isVisible` prop so refactoring shouldn't be too bad.